### PR TITLE
g.gui.example: fix broken module

### LINKED ISSUE #3232: [Bug] g.gui.example is broken (for many reasons)
**Describe the bug**

`g.gui.example` module does not work. Probably already for a long time. There are multiple reasons why:

* some instructions in `doc/gui/wxpython/example/README` on how to modify `Makefile` are missing and some are outdated
* `ExampleMapDialog.dataSizer.Add()` has no argument `item`
* `ExampleInfoTextManager.textCtrl` has no method `SetDefaultStyle`
* `dict.iteritems` does not exist in Py3
* `Example*Toolbar._getToolbarData()` uses wrong syntax

**To Reproduce**

Start `g.gui.example`.

**Expected behavior**

`g.gui.example` works.

### PR DESCRIPTION
**Pull request by pesekon2**
* fix instructions in g.gui.example README
* fix arguments in ExampleMapDialog.dataSizer.Add()
* ExampleInfoTextManager.textCtrl.SetDefaultStyle() -> SetFont()
* iteritems() -> items()
* _getToolbarData(): fix syntax

fix #3232

### CODE REVIEW

File: doc/gui/wxpython/example/README [hunk 1]
```diff
@@ -24,12 +24,15 @@ or it can be launched from the console.
...
  SRCFILES := $(wildcard icons/*.* scripts/* xml/*) \
--	$(wildcard core/* dbmgr/* gcp/* gmodeler/* ... \
-+	$(wildcard core/* dbmgr/* example/* gcp/* gmodeler/* ... \
+-       $(wildcard animation/*.py core/*.py datacatalog/*.py dbmgr/*.py gcp/*.py gmodeler/*.py \
++       $(wildcard animation/*.py core/*.py datacatalog/*.py dbmgr/*.py example/*.py gcp/*.py gmodeler/*.py \
 
--PYDSTDIRS := $(patsubst %,$(ETCDIR)/%,core dbmgr gcp gmodeler ... \
-+PYDSTDIRS := $(patsubst %,$(ETCDIR)/%,core dbmgr example gcp gmodeler ... \
+-PYDSTDIRS := $(patsubst %,$(DSTDIR)/%,animation core datacatalog dbmgr gcp gmodeler \
++PYDSTDIRS := $(patsubst %,$(DSTDIR)/%,animation core datacatalog dbmgr example gcp gmodeler \
```
Comment (wenzeslaus) [inline doc/gui/wxpython/example/README:35]: Let's avoid the need to keep this in sync by just saying what needs to be changed. SUBDIRS, SRCFILES, and PYDSTDIRS, or? I guess you need to add that only when you are adding another directory.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

File: doc/gui/wxpython/example/README (hunk 1 above)
Comment (pesekon2) [reply doc/gui/wxpython/example/README:35]: @wenzeslaus: What do you think then about changing this point to the following?

```
3. Edit ./gui/wxpython/Makefile:

SUBDIRS: Add example
SRCFILES: add example/*py
PYDSTDIRS: add example
```

Comment (landam) [reply doc/gui/wxpython/example/README:35]: @wenzeslaus Any feedback? Thanks.

Comment (wenzeslaus) [reply doc/gui/wxpython/example/README:35]: Yes, that looks like it addressed my comment.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
See my comment.

File: doc/gui/wxpython/example/README (hunk 1 above)
Comment (pesekon2) [reply doc/gui/wxpython/example/README]: @wenzeslaus: Thanks. Addressed in [384d3f3](https://github.com/OSGeo/grass/pull/3233/commits/384d3f39d45f6c8c99d16b6dd8b7bb85797ebd35)

**Review by wenzeslaus — APPROVED** [review]

File: doc/gui/wxpython/example/README (hunk 1 above)
Comment (wenzeslaus) [reply doc/gui/wxpython/example/README]: Thanks!

**Pull request merged**